packages/components/src/components/SquiggleViewer/ViewerProvider.tsx

```diff
@@ -293,39 +296,66 @@ export function useMergedSettings(path: SqValuePath) {
   return result;
 }
 
-export const ViewerProvider = forwardRef<
-  SquiggleViewerHandle,
-  PropsWithChildren<{
-    partialPlaygroundSettings: PartialPlaygroundSettings;
-    editor?: CodeEditorHandle;
-  }>
->(({ partialPlaygroundSettings, editor, children }, ref) => {
-  const [itemStore] = useState(() => new ItemStore());
-
-  useImperativeHandle(ref, () => ({
-    viewValuePath(path: SqValuePath) {
-      itemStore.scrollToPath(path);
-    },
-  }));
+type Props = PropsWithChildren<{
+  partialPlaygroundSettings: PartialPlaygroundSettings;
+  editor?: CodeEditorHandle;
+}>;
 
-  const [focused, setFocused] = useState<SqValuePath | undefined>();
+export const InnerViewerProvider = forwardRef<SquiggleViewerHandle, Props>(
+  (
+    { partialPlaygroundSettings: unstablePlaygroundSettings, editor, children },
+    ref
+  ) => {
+    const [itemStore] = useState(() => new ItemStore());
+
+    /**
+     * Because we often obtain `partialPlaygroundSettings` with spread syntax, its identity changes on each render, which could
```

Comment:
Why the "*" signs, when you already are using block comments?

Reply:
VS Code (probably Prettier, but I'm not sure) formats these better.

Like, it converts
```
/*
* hello
   * hi
*/
```

To:
```
/*
 * hello
 * hi
 */
```

on save.

Also, if you start it with `/**`, VS Code will autoformat it:
<img width="505" alt="Screenshot 2024-01-09 at 16 44 21" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/df01f1e2-e7f9-41fa-a916-5d8bec5f4dbc">

I'm still not sure if that's the best format, but it's the default for JS Doc: https://jsdoc.app/about-getting-started, and `starred-block` is the default in this ESLint rule: https://eslint.org/docs/latest/rules/multiline-comment-style